### DISCUSSION TITLE
How to enable retro in the game

### DISCUSSION THREAD

**Discussion by daxinhub** [EmulatorJS/EmulatorJS#888] [General]
![image](https://github.com/user-attachments/assets/b3f015b4-75ee-49f1-8465-3ce31347b0d2)
To enable the joystick, the game on this page failed to load and needs to be displayed by pressing the tab. But some emulators cannot appear, or can we modify the default joystick enabled in JS

**Comment by ethanaobrien** [comment]
Use the control settings menu?

**Comment by daxinhub** [comment]
> Use the control settings menu?

Yes, this is one of the solutions. But it can only be set, and it cannot directly switch out the left analog like retroarch (the left joystick and the direction keys are common), which is very useful. For example, I can enter the retroarch settings with tab in arcade, but I can't enter the retroarch settings in gba.
like the picture above

**Reply by ethanaobrien** [reply]
The analog stick controls are in the control settings menu if the system uses them. If not, they are hidden. If a system needs them that it’s not shown, you can add them by modifying emulator.js. 

I don’t have a plan to add what you’re requesting, since it’s already possible. Just not how you’d like for it to be done

**Reply by daxinhub** [reply]
> The analog stick controls are in the control settings menu if the system uses them. If not, they are hidden. If a system needs them that it’s not shown, you can add them by modifying emulator.js. 
> 
> 
> 
> I don’t have a plan to add what you’re requesting, since it’s already possible. Just not how you’d like for it to be done 

thank，I have found the relevant JS parameters, I just want the tab to switch to retroarch at any time

**Reply by ethanaobrien** [reply]
> I just want the tab to switch to retroarch at any time

While it is not supported, (meaning you will be on your own) see here https://github.com/EmulatorJS/EmulatorJS/issues/886#issuecomment-2309047599
